srcmedia/scss/search/_forms.scss

```diff
@@ -41,6 +41,33 @@
             border-bottom-style: solid;
         }
     }
+
+    input[type=number] {
+        appearance: textfield;
+        font-size: 0.8rem;
+        color: $black;
+        text-align: center;
+        border: 1px solid $light-grey;
+
+        // remove spinners in webkit browsers
```

Comment:
Should the comment note why we do this? There was a bug, right?

Reply:
yeah, iirc this was that whole thing mary found where clicking the down arrow on the spinner when it was at `min` would mysteriously "wrap"/reset it to the `max` value. i'll add a note